packages/graphql-hooks/rollup.config.js

```diff
@@ -1,3 +1,14 @@
+import babel from 'rollup-plugin-babel'
+import { sizeSnapshot } from 'rollup-plugin-size-snapshot'
+
 import generateRollupConfig from '../../config/rollup.config'
 
-export default generateRollupConfig('GraphQLHooks')
+export default generateRollupConfig('GraphQLHooks').concat({
+  input: 'src/middlewares/apqMiddleware.js',
+  output: {
+    file: `lib/middlewares/apqMiddleware.js`,
+    format: 'cjs',
+    indent: false
+  },
+  plugins: [babel(), sizeSnapshot()]
+})
```

Comment:
not sure what this is, mind clarifying?

Reply:
this makes sure that apqMiddleware is build separately from the main bundle (so no need to have it in the bundle if not used)